jani_generator/src/jani_generator/jani_entries/jani_constant.py

```diff
@@ -53,6 +54,7 @@ def jani_type_from_string(str_type: str) -> ValidTypes:
             raise ValueError(f"Type {str_type} not supported by Jani")
 
     # TODO: Move this to a util function file
+    @staticmethod
     def jani_type_to_string(c_type: ValidTypes) -> str:
```

Comment:
Do we have to add the Type here, too?
```suggestion
    def jani_type_to_string(c_type: Type[ValidTypes]) -> str:
```